appcues/src/main/java/com/appcues/ui/composables/AppcuesComposition.kt

```diff
@@ -156,9 +163,9 @@ private fun BoxScope.ComposeLastRenderingState(state: Rendering) {
 
 @Composable
 private fun rememberMetadataSettingTraits(
-    viewModel: AppcuesViewModel,
     metadataSettingTraits: MutableState<List<MetadataSettingTrait>>,
 ): MutableState<AppcuesStepMetadata>? {
+    val traitExceptionHandler = LocalAppcuesTraitExceptionHandler.current
```

Comment:
using this new trait exception handler, just a lambda that handles the exception, simplifies things so that the entire `AppcuesViewModel` does not need to be passed all the way down or used at all in testing scenarios.

Reply:
sure, I guess we had discussed this option when we decided to capture exception before